README.md

```diff
@@ -105,12 +105,19 @@ prototype-2	prototype-2-prototype	FALSE
 The sbt task `generatePackageLockReport` can produce a report of which Github repos deployed to Heroku contain a 
 `package.json` file and a `package-lock.json` file. To do this, the task will:
 1. Parse a local copy of `report.txt` generated by `generateHerokuReport` (meaning this task must be run as a prerequisite)
-1. Clone from Github to a local directory any repositories in the HMRC organisation that are named the same as a 
-   deployed Heroku prototype
-1. Produce a report of which deployed Heroku prototypes have a matching Github repository, and whether that repository 
-   contains a `package-lock.json` file.
-   
-The output will be saved to `package-local-analysis.txt`.
+2. For each deployed prototype, check if a Github repo exists with the same name as the Heroku app
+3. If not, try to resolve the Github repo name by searching for the commit ids of the 10 earliest Heroku releases of the app
+4. If the Github repo is known, check if it contains a package.json and/or package-lock.json
+5. Produce a report of deployed Heroku prototypes with the following details:
+* Heroku app name
+* Github repo name
+* If the repo has a package.json
+* If the repo has a package-lock.json
+* If the prototype is deployed to Heroku
+* If the prototype is running in Heroku
+* If the Github repo was found
+
+The output will be saved to `package-lock-analysis.tsv`.
 
 ```shell script
 sbt -Dheroku.apiToken=REPLACE_WITH_TOKEN "generateHerokuReport report.txt"
```

Comment:
@ellamdav does this script need a `-Dgithub.apiToken=REPLACE_WITH_GITHUB_TOKEN` too?

Reply:
doh!  yes, good spot - will add